packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapMarkerController.m

```diff
@@ -127,6 +129,10 @@ - (void)updateFromPlatformMarker:(FGMPlatformMarker *)platformMarker
   }
 
   [self setVisible:platformMarker.visible];
```

Comment:
Per investigation in https://github.com/flutter/packages/pull/7001, this should always be the last thing in these methods. We don't yet have a complete PR that tests and enforces that, but we should still try to not make it worse in the meantime.

Reply:
Moved the method call to last with comment (so that it is kept there in future changes as well).